### DISCUSSION TITLE
"Show more" is buggy on Update DNS record API reference doc

### DISCUSSION THREAD

**Discussion by chandru89new** [vercel/community#1048] [Help] [answered]
Link: https://vercel.com/docs/rest-api#endpoints/dns/update-an-existing-dns-record

If I click on the "Show more" toggle, whole screen goes blank/white and I can't really view more of the details for that endpoint/resource.

**Comment by chandru89new** [comment]
Additional details from the console: 
<img width="512" alt="image" src="https://user-images.githubusercontent.com/10336112/201537060-d4f816c7-b427-4026-96c9-cdc9e8653dcc.png">

**Comment by amyegan** [comment, marked as answer]
Thanks for reporting this! I have created an internal ticket to fix this bug.

**Reply by amyegan** [reply]
This bug was fixed today 🎉

**Reply by chandru89new** [reply]
Hurray. Thanks @amyegan !
